### DISCUSSION TITLE
[BUG] Is this a bug?

### DISCUSSION THREAD

**Discussion by ndcc5** [Chia-Network/chia-blockchain#3173] [General]
![HCK_}{P`Y`}PXYR(Y33JCLH](https://user-images.githubusercontent.com/6286019/115133172-c2199000-a038-11eb-9c7c-1c84b46541ec.png)
![~Y1@I%IX(45RX{XH8R399(M](https://user-images.githubusercontent.com/6286019/115133173-c3e35380-a038-11eb-98bc-a1a69bd548c2.png)

**Comment by Z-fly** [comment]
不是BUG吧

**Comment by ndcc5** [comment]
容量明明够，这是什么原因

**Comment by Billy-Hostage** [comment]
GiB单位和GB不同
1 GiB = 1.07374182 GB

**Comment by Z-fly** [comment]
> GiB单位和GB不同
> 1 GiB = 1.07374182 GB

硬盘厂原则上用1000MB表示1GB，但是win上是1024MB=1GB

**Comment by ndcc5** [comment]
那实际容量到多少才够429GIB?

**Comment by ndcc5** [comment]
465GB怎么算也够429.8GIB了吧

**Comment by jonny77** [comment]
你选择的时候,429G 需要1041的 临时空间.....

**Comment by ndcc5** [comment]
> 你选择的时候，429G需要1041的临时空间.....

了解了

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
